### ISSUE TITLE
tests/clean up

### ISSUE THREAD

**Issue by calvinmetcalf** [pouchdb/GQL#1]
we need to add tests, wrap it in a umd, and make it available at least via npm

**Comment by calvinmetcalf** [comment]
also pinging @neojski as this is his baby

**Comment by marfarma** [comment]
also needs to require collate -- and an example of how to add it to your project, because it's not documented anywhere.  Currently I've forked pouchdb and added it to the list of source files to include in the build.

I'm not clear how it's meant to work now that it's separated out into its own repository.  Will I still need to have a custom npm build of pouchdb in order to use this?

**Comment by calvinmetcalf** [comment]
You can actually include it via a script tag and it will work

**Comment by marfarma** [comment]
OK - I've forked it.  What can I use to model the testing setup after?  Do any of the other separated out plugins have tests I can copy to use as a starting point?  Since I'm using it in my project, I'm motivated to get this plugin released on npm or bower.

Unless @neojski has time to pick this up?

**Comment by marfarma** [comment]
collate seems the most complete.  Any reason not to copy over the setup from there - i.e. lib/index.js,  test/test.js, project.json with the test and build tasks, dist/pouchdb-gql.js, etc.?

**Comment by calvinmetcalf** [comment]
Yes look at the map reduce or spatial,  the mocha/chai/Istanbul set up
which they use is pretty good.
I can five you some help on Thursday until then you can copy the scripts
section of the geopouch or map reduce package.json and install the deps
with 'npm install --save-dev istanbul mocha chai'

**Comment by calvinmetcalf** [comment]
@MarFarMa I added some very basic test scaffolding

**Comment by marfarma** [comment]
Thanks.  That helps.
